releasenotes/releasenotes_develop.md

```diff
@@ -1,52 +1,299 @@
+  - Refinement of the sequence tile [2628](https://github.com/keptn/keptn/issues/2628)
+- SLI breakdown in table [2478](https://github.com/keptn/keptn/issues/2478)
+- Service screen refinement [3206](https://github.com/keptn/keptn/issues/3206)
+- Update doc references for 0.8.x [3205](https://github.com/keptn/keptn/issues/3205)
+- Misleading information in event stream of sequence screen [3016](https://github.com/keptn/keptn/issues/3016)
+- Show shipyard version in project tile [2909](https://github.com/keptn/keptn/issues/2909)
+- Sort the SLOs names in the Keptn Bridge Evaluation done results [1499](https://github.com/keptn/keptn/issues/1499)
+- Highlight stages more prominent in eventflow [2229](https://github.com/keptn/keptn/issues/2229)
+- Shows configuration-change events and its content [2872](https://github.com/keptn/keptn/issues/2872)
+- Shows evaluations for new `evaluation.finished` events (instead of `evaluation-done`) [2866](https://github.com/keptn/keptn/issues/2866) 
+- Use an HTTP-interceptor to add default headers and implement generic error handling [1987](https://github.com/keptn/keptn/issues/1987) 
+- Added COPY button for SLO content [1997](https://github.com/keptn/keptn/issues/1997)
+- *Refactoring*: Split project-board into smaller components [1989](https://github.com/keptn/keptn/issues/1989)
+- *Refactoring*: Create view-component for environments tab [2939](https://github.com/keptn/keptn/issues/2939)
+- *Refactoring*: Create view-component for integration tab [2942](https://github.com/keptn/keptn/issues/2942)
+- *Refactoring*: Create stage-overview component [2943](https://github.com/keptn/keptn/issues/2943)
+- *Refactoring*: Create stage-details component [2944](https://github.com/keptn/keptn/issues/2944)
+- *Refactoring*: Create view-component for sequences tab [2941](https://github.com/keptn/keptn/issues/2941)
+- *Refactoring*: Create view-component for services tab [2940](https://github.com/keptn/keptn/issues/2940)
+- *Fixed*: Manual approval does not trigger next task in sequence [3013](https://github.com/keptn/keptn/issues/3013)
+- *Fixed*: ERROR TypeError: this.data.configurationChange.values.image is undefined [3021](https://github.com/keptn/keptn/issues/3021)
+- *Fixed*: Approval not possible in cases when having the manual deployment strategy [2901](https://github.com/keptn/keptn/issues/2901)
+- *Fixed*: Keptn Bridge is not showing notification about the new Keptn version [2693](https://github.com/keptn/keptn/issues/2693)
+- *Fixed*: Keptn Bridge ignores deployed service artifact [2543](https://github.com/keptn/keptn/issues/2543)
 
 </p>
 </details>
 
+## Miscellaneous
+
+- Fixed several spelling mistakes [2849](https://github.com/keptn/keptn/issues/2849)
+- Format the Go imports [3150](https://github.com/keptn/keptn/issues/3150)
+- Test the linking of stages based on task sequence events: `sh.keptn.event.[stage].[sequence].finished` [2534](https://github.com/keptn/keptn/issues/2534)
```

Comment:
There is no issue on that, but we should address that we have updated many third-party-dependencies to their latest versions (see https://github.com/keptn/keptn/pulls?q=is%3Apr+author%3Aapp%2Fdependabot+is%3Amerged+ as a reference), most notably: 

**Go** (Microservices)

* google/uuid to 1.2.0
* go.mongodb.org/monto-driver to 1.4.6
* cloudevents/sdk-go (various versions needed)
* nats-io/nats-server/v2 to 2.1.9

**NodeJS** (Bridge)
* marked to 2.0.0
* higlights.js to 10.4.1

Reply:
I added it to the "miscellaneous" section